fx-src/FirefoxStorage.js

```diff
@@ -0,0 +1,484 @@
+import BaseAdapter from "../src/adapters/base";
+
+const { interfaces: Ci, results: Cr, utils: Cu } = Components;
+
+Cu.import("resource://gre/modules/Services.jsm");
+Cu.import("resource://gre/modules/FileUtils.jsm");
+
+function debug(message) {
```

Comment:
Consider what you want to do here other than use `dump`. There's `Log.jsm`, there's plain ol' `console.log`, …